lib/algebra.gi

```diff
@@ -2736,6 +2736,11 @@ InstallMethod( RadicalOfAlgebra,
     fi;
 
     n:= Dimension( A );
+
+    if n = 0 then
+      return A;
```

Comment:
Should it also set `ParentAttr` to `A`? It is bound in radicals returned by `SubalgebraNC( A, rad, "basis" )`.

Reply:
Setting a parent is useful only if exploiting the parent relation yields some information, thus setting an object to its own parent should be avoided.